protocols/swap/reports/SECURITY.md

```diff
@@ -1,175 +1,173 @@
-# Security Report: Swap
+# Swap: Security Report
 
-Smart Contract Security Report by Team Fluidity (team[at]fluidity[dot]io) and Phil Daian (feedback[at]stableset[dot]com)
-Hash of master used for report: [ef5cff0613532d27ecedb332e222ae0a75079841](https://github.com/airswap/airswap-protocols/commit/ef5cff0613532d27ecedb332e222ae0a75079841)
+Security report by Team Fluidity (team[at]fluidity[dot]io). Smart contracts are a nascent space, and no security audit procedure has been perfected. We welcome any suggestions and comments on this report, its contents, our methodology, or potential gaps in coverage.
 
-Swap [Source Code](https://github.com/airswap/airswap-protocols/tree/master/protocols/swap) and [README](../README.md)
+Swap [Source Code](https://github.com/airswap/airswap-protocols/tree/master/protocols/swap) and [README](../README.md) are available in this repository. Commit used for report: [2a83c1ff2e46e6befa45889aa556fdd31e5c71fb](https://github.com/airswap/airswap-protocols/commit/2a83c1ff2e46e6befa45889aa556fdd31e5c71fb)
 
 ## Introduction
 
-The Swap Protocol is a peer-to-peer protocol for trading Ethereum tokens that allows two parties to exchange tokens in an atomic transaction. It is a non-custodial exchange settlement contract. The new additions to the contract allow for trading of ERC-721 tokens, delegation for token trades to pre-authorized parties, and optionally distributing affiliate fees in tokens to a party who facilitates peers meeting. In addition, previous behavior of the AirSwap protocol is maintained in terms of swapping ERC-20 tokens. One significant change related to security is that the Swap protocol now does not deal with ether at all. The contracts are compiled with v0.5.10.a6ea5b19 (0.5.10 stable release).
+Swap is a non-custodial exchange settlement contract. It is used in the Swap Protocol, a peer-to-peer trading protocol for Ethereum tokens. The implementation supports trading ERC-20 and ERC-721 tokens. Additional features include the ability to authorize another party to sign or send orders on one's behalf, and the ability to optionally set an affiliate that is compensated for bringing together trading parties. This contract only transfers tokens, no raw ether (ETH). The following contracts are compiled with solidity 0.5.12.
 
 ## Structure
 
-The Swap contract is comprised a contract, an interface, and a library.
+Swap includes one contract, its interface, and their dependencies.
```

Comment:
Again I dont like `includes`. Swap doesnt include one contract, Swap IS that one contract.